### PR TITLE
Creation of the logspace entry

### LINKED ISSUE #5225: [Term Entry] PyTorch Tensors .logspace()
### Reason/inspiration (optional)

We would like a new entry on the `.logspace()` term under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/tensors/terms/logspace/logspace.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use
- A `Codebyte Example` section that provides a codebyte example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is an attempt to create the information on the logspace function for Pytorch. Hoping to help learners expand their knowledge on the subject.

### Issue Solved

Closes #5225 

### Type of Change

- Adding a new entry

### Checklist

- [/] All writings are my own.
- [/] My entry follows the Codecademy Docs style guide.
- [/] My changes generate no new warnings.
- [/] I have performed a self-review of my own writing and code.
- [/] I have checked my entry and corrected any misspellings.
- [/] I have made corresponding changes to the documentation if needed.
- [/] I have confirmed my changes are not being pushed from my forked `main` branch.
- [/] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [/] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/tensors/terms/logspace/logspace.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## Syntax
+
+```pseudo
+torch.logspace(start, end, steps, dtype=None)
+```
+
+- `start` is the base-10 logarithm of the first element in the tensor.
+- `end` is the base-10 logarithm of the last element in the tensor.
+- `steps` is the number of elements in the tensor.
+- `dtype` is the data type of the tensor.
```
Comment (cigar-galaxy82) [inline content/pytorch/concepts/tensors/terms/logspace/logspace.md:30]:
```suggestion
- `start`: The first number in the range expressed as a logarithm.
- `end`: The last number in the range expressed as a logarithm.
- `steps`: Number of elements to be returned in the tensor.
- `base`: The base of the logarithm used for calculating the values default value is 10.
- `dtype`: Specifies the data type of the returned tensor.
```

File: content/pytorch/concepts/tensors/terms/logspace/logspace.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+print(tensor)
+```
+
+Output:
+
+```shell
+tensor([1.0000, 3.1623, 10.0000], dtype=torch.float64)
+```
+
+In the first example, we created a tensor `tensor` containing 5 logarithmically spaced values between 1 and 1000 using the `.logspace()` function. The tensor `tensor` contains the values `[10.0000, 100.0000, 1000.0000, 10000.0000, 100000.0000]`.
```
Comment (cigar-galaxy82) [inline content/pytorch/concepts/tensors/terms/logspace/logspace.md:60]: Mention this after first example

File: content/pytorch/concepts/tensors/terms/logspace/logspace.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+In the second example, we created a tensor `tensor` containing 3 logarithmically spaced values between 0 and 1 using the `.logspace()` function with a data type of `torch.float64`. The tensor `tensor` contains the values `[1.0000, 3.1623, 10.0000]`.
```
Comment (cigar-galaxy82) [inline content/pytorch/concepts/tensors/terms/logspace/logspace.md:62]: Mention this after second example

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
Hi @arisdelacruz thank you for contributing to Codecademy docs, I have suggested a few changes please modify them.

**Comment by arisdelacruz** [comment]
> Hi @arisdelacruz thank you for contributing to Codecademy docs, I have suggested a few changes please modify them.

@cigar-galaxy82 

 done with the edits. Hope I hit them right. Thank you

**Review by cigar-galaxy82 — APPROVED** [review]

**Review by mamtawardhani — APPROVED** [review]
Thhank you for contributing @arisdelacruz 😄 

The entry looks good for a second round of review! 🚀
